### PR TITLE
AAP-3799 - Added Inclusive language statement to all documents

### PR DESCRIPTION
**Pull request by dcdacosta**
This PR adds the inclusive language module to all master.adoc files for the library. See https://issues.redhat.com/browse/AAP-3799.

Preview link (requires VPN): http://file.rdu.redhat.com/~ddacosta/aapinstall_3799/master.html
                                              http://file.rdu.redhat.com/~ddacosta/aapupgrade_3799/master.html
                                              http://file.rdu.redhat.com/~ddacosta/aaprn_3799/master.html

### CODE REVIEW

File: downstream/titles/aap-installation-guide/master.adoc [hunk 1]
```diff
@@ -14,6 +14,7 @@ Thank you for your interest in {PlatformName}. {PlatformNameShort} is a commerci
 
 This guide helps you to understand the installation requirements and processes behind installing {PlatformNameShort}. This document has been updated to include information for the latest release of {PlatformNameShort}.
  
+include::../../../common/making-open-source-more-inclusive.adoc[]
```
Comment (oraNod) [inline downstream/titles/aap-installation-guide/master.adoc:17]: Should create a symlink to the common folder in each directory so the include directive is like this: `include::common/making-open-source-more-inclusive.adoc[]`

The publishing system doesn't play nicely with `../` (although that might have changed).

Comment (oraNod) [reply downstream/titles/aap-installation-guide/master.adoc:17]: @dcdacosta Give me a shout if you want me to create the symlinks. I don't mind doing it. Also what do you think about changing `common` to `aap-common`?

**Review by ariordan-redhat — COMMENTED** [review]
@dcdacosta 

I can't build the docs because `context` isn't defined for `common/making-open-source-more-inclusive.adoc`

```
[id="making-open-source-more-inclusive_{context}"]
```

Do we need the context included for this module?

The build was successful for me when I deleted `_{context}` from the module ID (`[id="making-open-source-more-inclusive"]`)

It was also successful when I defined `context` before the `id=` line:

```
:context: common
[id="making-open-source-more-inclusive_{context}"]
```

One other point: The nesting in the index looks wrong in my build when the inclusive module is added:

image

Is there a way to change this?

**Comment by dcdacosta** [comment]
> @dcdacosta
> 
> I can't build the docs on my machine (a Mac) because `context` isn't defined for `common/making-open-source-more-inclusive.adoc`
> 
> ```
> [id="making-open-source-more-inclusive_{context}"]
> ```
> 
> Do we need the context included for this module?
> 
> The build was successful for me when I deleted `_{context}` from the module id line:
> 
> ```
> [id="making-open-source-more-inclusive"]
> ```
> 
> It was also successful when I defined `context` before the `id=` line:
> 
> ```
> :context: common
> [id="making-open-source-more-inclusive_{context}"]
> ```
> 
> One other point: The nesting in the index looks wrong in my build when the `common/making-open-source-more-inclusive.adoc` module is added:
> 
> common_module
> 
> Is there a way to change this?

Hi @ariordan-redhat My bad, I'll fix the context issue. I'm not sure how to fix the index issue but I'll put a shoutout in the CCS gchat space to see if anyone knows what I can do. Thanks for the feedback!

**Comment by oraNod** [comment]
What about creating an `assembly-aap.adoc` file that we set as the preface?

The assembly could include a brief concept module that explains AAP from a high level, the inclusive language initiative, and a pointer to the downloads page + any other standard content that could be included in every `master.adoc` file.

Here's an example for RHDG: https://access.redhat.com/documentation/en-us/red_hat_data_grid/8.3/html/red_hat_data_grid_8.3_release_notes/red-hat-data-grid

**Comment by ariordan-redhat** [comment]
There is a way to escape an asciidoc heading from being indexed, but I don't know whether the Pantheon build supports it:
```
[discrete]
= Making open source more inclusive
```
image


image

**Comment by oraNod** [comment]
Yes, Pantheon plays nicely with the `[discrete]` tag.

**Comment by dcdacosta** [comment]
Ah yes, I forgot about the discrete tag. I think I'll switch to using that for now and we can revisit the other options as a team later. Thanks @oraNod and @ariordan-redhat!

**Review by ariordan-redhat — CHANGES_REQUESTED** [review]
@dcdacosta 
I wrote a quick and clunky script to build all the docs. 
Everything built OK apart from the release notes; I added a suggestion for fixing this.

One doc (`analytics/automation-savings-planner/`)doesn't have a title in the first assembly (`analytics/assembly-automation-savings-planner.adoc`), so the nesting looked wrong. I'll take care of this in another PR: it's out of scope for this PR.

**Comment by oraNod** [comment]
Thanks for the work here @dcdacosta I'm going to close this PR as discussed to move the inclusive language statement to a dedicated branch for boilerplate and other common content that we can use across releases.

**Pull request closed without merging**